.github/workflows/test_development_environment.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, macOS-latest, windows-latest]
+        os: [ubuntu-latest, macOS-latest] # temporarily removed windows-latest as it seems to be unable to set token
```

Comment:
the latest windows os container seems to be unable to get npm token. have checked with windows user dickson to see that the repo is able to run fine on windows outside of the workflow

Reply:
UHH wait ah seems like i was able to get it to pass by running it once without cache and then running it again